doc/source/whatsnew/v0.12.0.rst

```diff
@@ -201,12 +201,12 @@ IO enhancements
     You can use ``pd.read_html()`` to read the output from ``DataFrame.to_html()`` like so
 
     .. ipython:: python
-       :okwarning:
 
+        import io
         df = pd.DataFrame({"a": range(3), "b": list("abc")})
         print(df)
         html = df.to_html()
-        alist = pd.read_html(html, index_col=0)
+        alist = pd.read_html(io.StringIO(html), index_col=0)
```

Comment:
as this is such an old whatsnew is is possible to not change this (other than making this not run the code?)

Reply:
Yeah the `:okwarning:` was supressing the deprecation warning, so the only other way would be to change to a `code-block`